bigchaindb/commands/bigchaindb.py

```diff
@@ -153,7 +156,6 @@ def run_init(args):
         print('The database already exists.', file=sys.stderr)
         print('If you wish to re-initialize it, first drop it.', file=sys.stderr)
 
-
```

Comment:
Those `\n\n` are there because we follow [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines).